subsys/dm/Kconfig

```
@@ -15,32 +15,41 @@ if DM_MODULE
 
 menu "nRF DM (Distance Measurement)"
 
-choice NRF_DM_TIMER
-	default NRF_DM_TIMER2
-	prompt "Timer to use for the nrf_dm library"
+choice DM_TIMER
+	default DM_TIMER2
+	prompt "Timer to use for the nrf_dm library."
 
-config NRF_DM_TIMER0
+config DM_TIMER0
 	depends on HAS_HW_NRF_TIMER0
 	bool "TIMER0"
 
-config NRF_DM_TIMER1
+config DM_TIMER1
 	depends on HAS_HW_NRF_TIMER1
 	bool "TIMER1"
 
-config NRF_DM_TIMER2
+config DM_TIMER2
 	depends on HAS_HW_NRF_TIMER2
 	bool "TIMER2"
 
-config NRF_DM_TIMER3
+config DM_TIMER3
 	depends on HAS_HW_NRF_TIMER3
 	bool "TIMER3"
 
-config NRF_DM_TIMER4
+config DM_TIMER4
 	depends on HAS_HW_NRF_TIMER4
 	bool "TIMER4"
 
 endchoice
 
+config DM_USE_HIGH_PRECISION_CALC
```

I think it is better to call this option/feature like this

```suggestion
config DM_HIGH_PRECISION_CALC
```

The verb is silently implied in this Kconfig option name. For example,  the ```config BT``` enables the Bluetooth subsystem in Zephyr (it is not called ```config BT_ENABLE```). Similarly here, I think that the `use` or `enable` verb is silently implied in the `DM_HIGH_PRECISION_CALC` configuration name.

---

Good point :+1: